requirements-optional.txt

```diff
@@ -1,4 +1,4 @@
 matplotlib
 pyrevolve==2.2.4
-scipy
+scipy>=1.13.0
```

Comment:
I think it's too restrictive, it's a very new version, would wrap the import in try/except to handle different version import path

Reply:
ah sorry jsut saw that